WordPress/src/main/java/org/wordpress/android/ui/bloggingreminders/EpilogueBuilder.kt

```diff
@@ -0,0 +1,87 @@
+package org.wordpress.android.ui.bloggingreminders
+
+import android.os.Build.VERSION_CODES
+import androidx.annotation.RequiresApi
+import org.wordpress.android.R.drawable
+import org.wordpress.android.R.string
+import org.wordpress.android.fluxc.model.BloggingRemindersModel
+import org.wordpress.android.ui.bloggingreminders.BloggingRemindersItem.Caption
+import org.wordpress.android.ui.bloggingreminders.BloggingRemindersItem.HighEmphasisText
+import org.wordpress.android.ui.bloggingreminders.BloggingRemindersItem.Illustration
+import org.wordpress.android.ui.bloggingreminders.BloggingRemindersItem.Title
+import org.wordpress.android.ui.bloggingreminders.BloggingRemindersViewModel.UiState.PrimaryButton
+import org.wordpress.android.ui.utils.ListItemInteraction
+import org.wordpress.android.ui.utils.UiString.UiStringRes
+import org.wordpress.android.ui.utils.UiString.UiStringResWithParams
+import org.wordpress.android.ui.utils.UiString.UiStringText
+import java.time.DayOfWeek
+import java.time.format.TextStyle
+import java.util.Locale
+import javax.inject.Inject
+
+class EpilogueBuilder @Inject constructor(
+    private val dayLabelUtils: DayLabelUtils
+) {
+    @RequiresApi(VERSION_CODES.O) // TODO: Remove this annotation once de-sugaring PR is merged
+    fun buildUiItems(
+        bloggingRemindersModel: BloggingRemindersModel?
+    ): List<BloggingRemindersItem> {
+        val numberOfTimes = dayLabelUtils.buildNTimesLabel(bloggingRemindersModel)
+        val enabledDays = bloggingRemindersModel?.enabledDays?.map { DayOfWeek.valueOf(it.name) }
+
+        val selectedDays = when (enabledDays?.count()) {
+            ONE_DAY -> enabledDays.joinToString { formattedDay(it) }
+            TWO_DAYS -> enabledDays.joinToString(separator = " and ") { formattedDay(it) }
```

Comment:
Is this hardcoded `" and "` intentional? I suppose we should have a translatable string here instead at least?

Reply:
all this should be replaced once formatting options are available after merging desugaring